src/filelock/_api.py

```diff
@@ -116,6 +116,7 @@ def acquire(
         poll_interval: float = 0.05,
         *,
         poll_intervall: float | None = None,
+        blocking=True,
```

Comment:
A better solution wouldn't just use the value 0 to mean non-blocking? Do we need this extra argument?

Reply:
As discussed in the issue #57, a timeout value of 0 currently means poll-indefinitely. Adding an additional argument makes it's functionality more clear without breaking backwards compatibility.